samples/drivers/sdhc/sdhc_read_write/README.rst

```diff
@@ -0,0 +1,44 @@
+.. _sdhc:
+
+SDHC CMD52 Read/Write Sample
+============================
+
```

Comment:
needs to use the `.. zephyr:code-sample::` directive. Please check other samples for... examples, and check template as well https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/templates/sample.tmpl with some guidelines w.r.t wording, heading levels to use , etc.
Also, no need to have "sample" in the name